### PR TITLE
Implement composer post-install hook to create log file

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <patrick@puiterwijk.org>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
         "squizlabs/php_codesniffer": "3.0.0RC3"
     },
     "scripts": {
+        "post-install-cmd": [
+            "touch anime/Services/error.log",
+            "if [ ! -f anime/Services/state.json ]; then echo '{}' >anime/Services/state.json; fi"],
```
Comment (beverloo) [inline composer.json]: This probably needs a small PHP script— neither of these lines will work on Windows :(

Comment (puiterwijk) [inline composer.json]: Windows is a target platform? That's somewhat disappointing :(.
But understood.

Comment (beverloo) [inline composer.json]: I use all three major platforms daily, sorry :P

**Comment by puiterwijk** [comment]
Re-implemented this in native PHP :-)

File: anime/Services/Utils.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+// Copyright 2017 Peter Beverloo. All rights reserved.
+// Use of this source code is governed by the MIT license, a copy of which can
+// be found in the LICENSE file.
+
+declare(strict_types=1);
+
+namespace Anime\Services;
+
+use Composer\Script\Event;
```
Comment (beverloo) [inline anime/Services/Utils.php]: `Event` seems to be unused

File: composer.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
         "squizlabs/php_codesniffer": "3.0.0RC3"
     },
     "scripts": {
+        "post-install-cmd": "Anime\\Services\\Utils::postInstall",
```
Comment (beverloo) [inline composer.json]: Cool, I didn't know this syntax worked :)

**Comment by puiterwijk** [comment]
I don't seem to actually have write access to the repo though :)
(Not complaining, just pointing out)

**Pull request merged**